src/GameMain/ScoreBoard.java

```diff
@@ -68,9 +68,93 @@ public ScoreBoard(GameView theView) {
         restartButton.addActionListener(this);
     }
 
+    @Override
+    public void paintComponent(Graphics g) {
+        // Loading screen
+        loadScore();
+        // Getting board stage
+        stage = Board.getStage();
+
+        super.paintComponent(g);
+        // Loading of font
+        Font font = loadFont();
+
+        ArrayList<Image> tankList = new ArrayList<>(
+                Arrays.asList(
+                        imageInstance.getTankBasic(),
+                        imageInstance.getTankFast(),
+                        imageInstance.getTankPower(),
+                        imageInstance.getTankArmor()
+                )
+        );
+
+        g.setFont(font);
+        g.setColor(Color.WHITE);
+        g.drawString("STAGE   " + String.valueOf(stage), 97 + SHIFT, 60);
+
+        g.setColor(Color.RED);
+        g.drawString("1-PLAYER", 37 + SHIFT, 95);
```

Comment:
Strings like "STAGE," "1-PLAYER," and color values are hardcoded. Using constants or externalizing these values would make the code more maintainable and easier to modify in the future.